docs/client_reference.rst

```diff
@@ -88,14 +88,14 @@ The client session supports the context manager protocol for self closing.
       that generation. Note that ``Content-Length`` autogeneration can't
       be skipped.
 
-      Iterable of :class:`str` or :class:`~aiohttp.istr` (optional)
+      Iterable of :class:`str` or :class:`~multidict.istr` (optional)
 
    :param aiohttp.BasicAuth auth: an object that represents HTTP Basic
                                   Authorization (optional)
 
    :param version: supported HTTP version, ``HTTP 1.1`` by default.
 
-   :param cookie_jar: Cookie Jar, :class:`AbstractCookieJar` instance.
+   :param cookie_jar: Cookie Jar, :class:`aiohttp.abc.aiohttp.abc.AbstractCookieJar` instance.
```

Comment:
This prefix is wrong as discussed in another PR.

Reply:
I have warning with `aiohttp.abc.AbstractCookieJar` and `AbstractCookieJar`. with a double prefix I don't have a warning. If you insist, I'll leave it as it is

Reply:
You need to fix this warning by removing the prefix from the respective `.. class::` definition. It's only duplicated because it's both in the class definition and `.. currentmodule::` at the top of the document — they are merged.

Reply:
I try it later